js/generate.js

```diff
@@ -214,13 +221,19 @@ function fillTables() {
 			}
 			fillCell(el8, data.tests);
 			el8.classList.add("tests");
-                        if (counterReq == counterRes) {
+		        });
+                    };
+                }
+                var p = fetch("data/" + spec + ".json").then(
```

Comment:
Thank you so much Marcos. What does the text say?

Sent by Outlook<http://taps.io/outlookmobile> for Android



On Sun, Feb 14, 2016 at 7:15 PM -0800, "Marcos Caceres" <notifications@github.com<mailto:notifications@github.com>> wrote:


In js/generate.js<https://github.com/w3c-webmob/mobile-web-app-standards/pull/51#discussion_r52856306>:

> @@ -214,13 +221,19 @@ function fillTables() {
>                       }
>                       fillCell(el8, data.tests);
>                       el8.classList.add("tests");
> -                        if (counterReq == counterRes) {
> +                     });
> +                    };
> +                }
> +                var p = fetch("data/" + spec + ".json").then(


nit, break on .then() here too.

-
Reply to this email directly or view it on GitHub<https://github.com/w3c-webmob/mobile-web-app-standards/pull/51/files#r52856306>.